example/server/index.js

```diff
@@ -58,9 +58,25 @@ app.get('/token', (req, res) => {
       res.json(apiRes.data);
     })
     .catch((error) => {
-      console.error(error);
+      if (error.response) {
+        // The request was made and the server responded with a status code
+        // that falls out of the range of 2xx
+        console.log(error.response.data);
```

Comment:
Nit: can we use warn or error or info instead of log? Prefer to generally avoid log when possible to make the intent more clear.